### PR TITLE
Fix phpdoc (missing brackets)

### PR DESCRIPTION
**Pull request by bytehead**

### CODE REVIEW

File: tests/Contao/ValidatorTest.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function testValidatesEmailAddresses(string $email, $expected): void
     }
 
     /**
-     * @return (string,int)[]
+     * @return array(string,int)[]
```
Comment (ausi) [inline tests/Contao/ValidatorTest.php]: I think this should be `(string,int)[][]`?

File: tests/EventListener/StoreRefererListenerTest.php [hunk 2]
```diff
@@ -60,7 +60,7 @@ public function testStoresTheReferer(Request $request, ?array $currentReferer, ?
     }
 
     /**
-     * @return array<string,(Request|array<string,string>[]|null)[]>
+     * @return array<string,(Request|array<string,string>|null)>[]
```
Comment (ausi) [inline tests/EventListener/StoreRefererListenerTest.php]: Shouldn’t this be `array<string,(Request|array|null)[]>`?

File: tests/Contao/ValidatorTest.php (hunk 1 above)
Comment (bytehead) [reply tests/Contao/ValidatorTest.php]: will change it to `(string|int)[][]`

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**